docs/content/release_notes/whats_new.md

```diff
@@ -8,6 +8,15 @@ IBM® block storage CSI driver 1.9.0 introduces the enhancements that are detail
 
 Stretched volumes and stretched snapshots (FlashCopy) are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
 
+## New Call Home support
+
+IBM® block storage CSI driver 1.9.0 now supports using the storage system's built-in Call Home feature. The CSI driver volume identifier is the `_pvc-` tag.
```

Comment:
I'm not sure `_pvc-` is a tag. maybe a substring?
or something like:
```suggestion
IBM® block storage CSI driver 1.9.0 now supports using the storage system's built-in Call Home feature. The CSI driver volumes are identified by containing `_pvc-`.
```
though IMO this is an implementation detail the users don't need to know about

Reply:
@oriyarde  IMO I think that it's important at this stage to show how CSI uses the Call Home even in this state, so that there is some context as to why/how it's used.
What about -
"The CSI driver volumes are identified by containing the `_pvc-` identifier within the volume name."
or
"The CSI driver volumes contain the  `_pvc-` identifier."

Reply:
I'm just not sure if `_pvc-` can really be considered an identifier, even if we use it in order to identify our driver.
perhaps:
"The CSI driver volumes are identified by containing the `_pvc-` substring within the volume name."
or even better:
"The CSI driver volumes are identified by containing `_pvc-` in their name."

Reply:
done

Reply:
got it. thanks. updated